### DISCUSSION TITLE
Integrator for laplacian plus constant

### DISCUSSION THREAD

**Discussion by nilsfriess** [mfem/mfem#4085] [Q&A] [answered]
Hi, I'm currently playing around with mfem and right now I'm trying to setup a simple FE discretisation for the differential operator `-Delta u + k^2 I`, where `I` is the identity. What would be the best approach to create a `mfem::BilinearForm` for this operator?

**Comment by tzanio** [comment, marked as answer]
```c++
   double k = 4.0;

   ConstantCoefficient one(1.0);
   ConstantCoefficient kappa2(k*k);

   a.AddDomainIntegrator(new DiffusionIntegrator(one));
   a.AddDomainIntegrator(new MassIntegrator(kappa2));
```

**Reply by nilsfriess** [reply]
Thanks.
